src/slashCommand/slashCommand.interface.ts

```diff
@@ -1,8 +1,11 @@
-import { SlashCommandBuilder } from 'discord.js';
+import { ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js';
 
 export interface BaseCommand {
+  // TODO enforce static method implementation in commands with an abstract class?
   slashCommandBuilder: Omit<
     SlashCommandBuilder,
     'addSubcommand' | 'addSubcommandGroup'
   >;
+
+  handleCommand(arg: ChatInputCommandInteraction): Promise<any>;
```

Comment:
Looks like we could make this `Promise<void>` for now?

Reply:
Agreed. I chose "any" to leave it open, but since we don't do anything with a potential return value, it can be void for now.